tests/test_keyfile_creation.py

```diff
@@ -23,6 +23,18 @@ def test_pbkdf2_keyfile_creation():
     assert derived_private_key == PRIVATE_KEY
 
 
+def test_pbkdf2_keyfile_salt32_creation():
+    keyfile_json = create_keyfile_json(
+        PRIVATE_KEY,
+        password=PASSWORD,
+        kdf='pbkdf2',
+        iterations=1,
+        salt_size=32
+    )
+    derived_private_key = decode_keyfile_json(keyfile_json, PASSWORD)
+    assert derived_private_key == PRIVATE_KEY
```

Comment:
What do you think about adding an assertion in here about the actual size of the salt in the JSON document?